### ISSUE TITLE
  Canceling commands [BUG]? - 1.18.2

### ISSUE THREAD

**Issue by qVoidreaver** [Cyber2049/Epic-Fight---Indestructible#12]
First of all, I'd like to know if it's a bug in your mod or in Epic Fight itself. It's very difficult to get in touch with Epic's creators, so I thought I'd look you up and maybe you could shed some light on this problem.

 The problem is that it's canceling the "playsound" command (actually, it could be canceling all of them, but my problem is with playsound). "How do you cancel playsound?" This command is canceled if I attack at the moment of the animation that activates it or even during that animation.

I particularly have no idea if this is a problem caused by your mod or by epicfight, because I notice that, when I create a creature model using epicfight (not using the advanced mobptach, of course) it happens that, if I end up attacking a mob during its attack, the next one comes right after and sometimes even very quickly (as if canceling some kind of animation).

 Oh, and yes, this has nothing to do with "behavior_series", it doesn't change anything if I change anything from there, whether it's "looping" or "canBeInterrupted"

**Comment by austinsupercar** [comment]
I don't think this is a bug because when I disable short and long stun animations the commands don't get interrupted unless the target is neutralized or knocked down. The tutorial specifically says if the animation is interrupted so is the command also gets interrupted, removing the short stun animation from your mob patch file should fix this because it stops the entity from getting stunned by regular attacks (if you want special attacks but not regular attacks to interrupt the commands/animations keep the long stun animation defined and just remove the short stun animation).

**Comment by austinsupercar** [comment]
Alternatively might also be able to give your entity stun immunity 0.01 seconds (less than a tick) after the animation that triggers the commands is triggered to make it impossible for the player to stun the entity.
